pyglmnet/pyglmnet.py

```diff
@@ -432,8 +432,27 @@ def fit_predict(self, X, y):
         """
         return self.fit(X, y).predict(X)
 
-    def score(self, y, yhat, ynull=None, method='deviance'):
-        """Score the model.
+    def score(self, X, y):
+        """Scoring function compatible with sklearn.
```

Comment:
but there isn't an argument to expose this ... I agree it could be anything else but we should make our best efforts to document what the function already does